src/Symfony/Component/Serializer/Normalizer/CacheableSupportsMethodInterface.php

```diff
@@ -19,6 +19,7 @@
  * supports*() methods will be cached by type and format.
  *
  * @author Kévin Dunglas <dunglas@gmail.com>
+ * @deprecated since 6.3, to be removed in 7.0, implements `SupportedTypesMethodInterface` instead
```

Comment:
we also need to deprecate each `hasCacheableSupportsMethod` methods individually